cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,332 @@
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+resource "cockroach_client_ca_cert" "yourclustername" {
+  id = cockroach_cluster.example.id
+  x509_pem_cert = file("cockroach_client_ca_cert.json")
+}
+~~~
+</section>
+
+## Delete the certificate authority (CA) certificate for a dedicated cluster
+
+Remove the configured CA certificate from the cluster. Clients will no longer be able to authenticate with certificates signed by this CA certificate.
+
+{{site.data.alerts.callout_success}}
+Managing the certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="api">Using the API</button>
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request DELETE \
+  --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+  --header "Authorization: Bearer ${API_KEY}"
+~~~
+
+~~~txt
+200 OK
+~~~
+</section>
+<section class="filter-content" markdown="1" data-scope="tf">
+
+
+To delete the client CA cert on a cluster, remove the `cockroach_client_ca_cert` resource from your terraform configuration, then run `terraform apply`.
+</section>
+
+## Authenticate a SQL client to a {{ site.data.products.dedicated }} cluster
+
+To use certificate authentication for a SQL client, you must include the filepaths to the client's private key and public certificate. The public certificate must be signed by a certificate authority (CA) who's public certificate is configured to be trusted by the cluster. Refer to [Upload a certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster](#upload-a-certificate-authority-ca-certificate-for-a-cockroachdb-dedicated-cluster)
+
+1. From your cluster's overview page, `https://cockroachlabs.cloud/cluster/{ your cluster ID }`, click on the **Connect** button.
+
+1. Execute the command listed under **Download CA Cert** to download the required public certificate. This is used by your client to verify the identity of the cluster.
+
+1. Obtain a connection string/command for your cluster. This connection string is designed for password authentication and must be modified.
+
+    1. Remove the placeholder password from the connection string.
+
+    1. Construct the full connection string by providing the paths to `sslrootcert` (the cluster's public CA certificate), `sslcert` (the client's public certificate, which must be signed by the same CA), and `sslkey` (the client's private key).
```

Comment:
>`sslcert` (the client's public certificate, which must be signed by the same CA)

Maybe be explicit here and say "signed by the client's CA".  "Same CA" may sound like we're talking about the cluster CA?